kwok/README.md

```diff
@@ -38,12 +38,17 @@ spec:
           operator: In
           values: ["spot"]
       nodeClassRef:
-        name: nil
+        name: default
```

Comment:
We're going to be wanting this to have all the detail at v1. Better to just add it in there now so that we are adhering to best practices?